iaac/terraform/kubernetes-addons/amazon-controller-for-k8s-sagemaker/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_iam_policy" "sagemaker_ack_controller_studio_access" {
+  name_prefix        = "${local.service}-ack-controller-policy"
+  description = "IAM policy for the ${local.service} ack controller"
+  policy        = "${file("${path.module}/sagemaker_studio_access.json")}"
+}
+
+module "irsa" {
+  source            = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/irsa?ref=v4.9.0"
+  kubernetes_namespace = local.namespace
+  create_kubernetes_namespace = true
+  create_kubernetes_service_account = false
```

Comment:
false?

Reply:
sa created by the helm chart, will conflict otherwise